src/test/java/com/github/piedpiper/node/stepfunctions/StepFunctionsExecuteTest.java

```diff
@@ -0,0 +1,157 @@
+package com.github.piedpiper.node.stepfunctions;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Date;
+import org.mockito.Mock;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.amazonaws.services.stepfunctions.AWSStepFunctions;
+import com.amazonaws.services.stepfunctions.AWSStepFunctionsClientBuilder;
+import com.amazonaws.services.stepfunctions.model.StartExecutionRequest;
+import com.amazonaws.services.stepfunctions.model.StartExecutionResult;
+import com.github.commons.utils.JsonUtils;
+import com.github.piedpiper.common.PiedPiperConstants;
+import com.github.piedpiper.node.NodeInput;
+import com.github.piedpiper.node.NodeOutput;
+
+
+import org.junit.Assert;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({AWSStepFunctionsClientBuilder.class,StartExecutionRequest.class})
+public class StepFunctionsExecuteTest {
+		
+	@Mock
+	private AWSStepFunctionsClientBuilder sfClientBuilder;
+	
+	@Mock
+	private AWSStepFunctions sfClient;
+	
+	@Mock
+	private StartExecutionRequest sfExecuteRequest;
+	
+
+	
+	
+	
+	@Test
+	@Before
+	public void builderSetup() throws Exception{
+		PowerMockito.mockStatic(AWSStepFunctionsClientBuilder.class);
```

Comment:
If this is coming from guice u can build a mock module and avoide this mockStatic